### PR TITLE
feat(webapp): load chart data by default

### LINKED ISSUE #70: Total income graph
Area chart depicting total income over time for all delegates in USD and EOS 

image

### LINKED ISSUE #88: select all in graph filter
As a guest user I want the ability to select all elections and all years to see an aggregate of all the data 

image

The all selection should be the default view so that users can see data as soon as they open the page

i user selects all they get total income per election

### LINKED ISSUE #100: Load chart data by default
As a user i want to see the latest chart data as soon as i enter the income page

### LINKED ISSUE #106: Create view by total incomes on elections
Create on hasura a view to get the total amount and usd_total on incomes from all elections and divided by election

### PR DESCRIPTION
**Pull request by warnerHurtado**
### What does this PR do?

- Resolve #100 
- Resolve #88 
- Resolve #70
- Resolve #106 

### Steps to test

1. Run the project
2. Open in your browser localhost:3000
3. Click on income in the sidebar
4. See income options

#### CheckList

- [X] Follow proper Markdown format
- [X] The content is adequate
- [X] The content is available in both english and spanish
- [X] I Ran a spell check

### CODE REVIEW

**Review by leisterfrancisco — CHANGES_REQUESTED** [review]
Take a look at the warning

File: hasura/metadata/databases/default/tables/public_total_incomes_by_election.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+table:
+  name: total_incomes_by_election
```
Comment (xavier506) [inline hasura/metadata/databases/default/tables/public_total_incomes_by_election.yaml:2]: table names should be singular `total_income_by_election`

File: webapp/src/language/en.json [hunk 2]
```diff
@@ -40,6 +40,7 @@
     "textCurrencySelect": "Currency",
     "textDelegateSelect": "Delegate",
     "titleTable": "Delegates ranked in order of income from highest to lowest",
+    "titleTable2": "Elections ranked in order of election from highest to lowest",
```
Comment (xavier506) [inline webapp/src/language/en.json:43]: Elections sorted from newest to oldest

File: webapp/src/language/es.json [hunk 3]
```diff
@@ -40,6 +40,7 @@
     "textCurrencySelect": "Moneda",
     "textDelegateSelect": "Delegado",
     "titleTable": "Delegados ordenados por ingresos de mayor a menor",
+    "titleTable2": "Elecciones ordenadas de mayor a menor",
```
Comment (xavier506) [inline webapp/src/language/es.json:43]: Elecciones ordenadas de mas reciente a mas vieja?

**Review by xavier506 — APPROVED** [review]
LGTM

**Pull request merged**